### PR TITLE
feat(atom/tag): added new small tag size

### PR DESCRIPTION
**Pull request by beatrizip**
Added 'small' and 'medium' classes and style

### CODE REVIEW

File: components/atom/tag/src/index.js [hunk 1]
```diff
@@ -5,12 +5,25 @@ import classnames from 'classnames'
...
+   * @return {string}
+   */
   get _classNames() {
-    const {className, icon} = this.props
-    return classnames('sui-AtomTag', className, icon && 'sui-AtomTag-hasIcon')
+    const {className, icon, size} = this.props
+    return classnames(
+      'sui-AtomTag',
+      `sui-AtomTag-${size}`,
+      className,
```
Comment (jcger) [inline components/atom/tag/src/index.js]: As you are working on this component, could you please remove the className prop completely as we don't want to make components "styleable" from outside?

**Review by jcger — APPROVED** [review]

File: components/atom/tag/src/index.scss [hunk 2]
```diff
@@ -83,4 +84,18 @@ $p-atom-tag: 0 $p-l !default;
       background-color: $c-primary-dark;
     }
   }
+
+  &-small {
+    height: $p-xl;
```
Comment (midudev) [inline components/atom/tag/src/index.scss]: I'm not sure if we should use here `$p-xl` when the default size is using a `$h-atom-tag`. So, while the default one is customizable via local variables to the component, the small size is customizable using global ones. We might want to use a multiplier here in order to create the small using `$h-atom-tag * 0.8` for example.

The same happen with the line-height and the padding.

Other solution could be having also component variables for the small size.

File: components/atom/tag/src/index.js (hunk 1 above)
Comment (davidbarna) [reply components/atom/tag/src/index.js]: Aren't we making a breaking change removing className? @jcger

Comment (jcger) [reply components/atom/tag/src/index.js]: Being strict we should create a major, but it looks like there is nobody (except @beatrizip) using it

https://www.npmjs.com/package/@schibstedspain/sui-atom-tag
https://github.schibsted.io/search?q=sui-atom-tag&type=Code&utf8=✓

File: components/atom/tag/src/index.scss (hunk 2 above)
Comment (beatrizip) [reply components/atom/tag/src/index.scss]: ok! tx!

**Review by midudev — APPROVED** [review]

File: components/atom/tag/src/index.scss [hunk 3]
```diff
@@ -22,14 +24,14 @@ $p-atom-tag: 0 $p-l !default;
 
 .sui-AtomTag {
   background-color: $bgc-atom-tag;
-  border-radius: ceil($h-atom-tag / 2);
+  border-radius: ceil($h-atom-tag-medium / 2);
```
Comment (davidbarna) [inline components/atom/tag/src/index.scss]: are convention for "medium" is "m"

File: components/atom/tag/src/index.scss [hunk 4]
```diff
@@ -2,15 +2,17 @@
 
 $bgc-atom-tag: color-variation($c-gray, 3) !default;
 $c-atom-tag-actionable: $c-white !default;
-$h-atom-tag: $p-xxl !default;
+$h-atom-tag-medium: 32px !default;
+$h-atom-tag-small: 24px !default;
```
Comment (davidbarna) [inline components/atom/tag/src/index.scss]: our convention for "small" is "s"

File: components/atom/tag/src/index.scss [hunk 5]
```diff
@@ -2,15 +2,17 @@
 
 $bgc-atom-tag: color-variation($c-gray, 3) !default;
 $c-atom-tag-actionable: $c-white !default;
-$h-atom-tag: $p-xxl !default;
+$h-atom-tag-medium: 32px !default;
```
Comment (davidbarna) [inline components/atom/tag/src/index.scss]: I'm not sure if it should be the case here but for atom-button we expressed sizes with paddings. https://github.com/SUI-Components/sui-components/blob/master/components/atom/button/src/index.scss#L14

The question underneath is "if the paddings grow, should tags vertical padding grow as well?". I'd say  "yes". Sizes of tags and buttons are used to mark space between text and border: a padding. So the height should be expressed in paddings and the text vertically centered. It's my opinion.

Comment (jcger) [reply components/atom/tag/src/index.scss]: I agree with you, I'm gonna talk to the UI guys

File: components/atom/tag/src/index.scss (hunk 3 above)
Comment (beatrizip) [reply components/atom/tag/src/index.scss]: 👍

File: components/atom/tag/src/index.scss (hunk 4 above)
Comment (beatrizip) [reply components/atom/tag/src/index.scss]: 👍

File: components/atom/tag/src/index.scss (hunk 5 above)
Comment (beatrizip) [reply components/atom/tag/src/index.scss]: After the conversation the agreement is to use the height of the component

**Review by PablitoGS — APPROVED** [review]

**Review by nguasch — APPROVED** [review]

**Pull request merged**